cmd/initializer/initializer.go

```diff
@@ -1,13 +1,21 @@
 package main
 
 import (
+	goflag "flag"
 	"fmt"
 	"os"
 
+	"github.com/golang/glog"
```

Comment:
i think k/k switches to klog

Reply:
glog is already in the code base. We'll do the switch in one PR. Created https://github.com/kubernetes-sigs/kube-storage-version-migrator/issues/15 to track.